test/compute/ec2.js

```diff
@@ -0,0 +1,155 @@
+const chai = require("chai");
+const assert = chai.assert;
+const providers = require("../../lib/providers");
+const nodeCloud = require("../../lib");
+const nock = require("nock");
+
+const ncAWS = nodeCloud.getProvider(
+  providers.AWS,
+  "/home/rajika/projects/gsoc/config.json"
+);
+const options = {
+  apiVersion: "2016-11-15"
+};
+const ec2 = ncAWS.EC2(options);
+
+describe("AWS EC2", () => {
+  before(() => {});
+
+  it("should list all EC2 instance", done => {
+    nock("https://ec2.us-west-2.amazonaws.com").post("/").reply(200, {
+      Reservations: [
+        {
+          ReservationId: "r-0df3e7df18e0db8e3",
```

Comment:
Since we're using `nock` to mock responses from `AWS`, I have added an Id generated by AWS.